cloud-logging/README.md

```diff
@@ -1,6 +1,7 @@
 # Cloud Logging
```

Comment:
The README changes needs to be done to the [Cloud Shell Tutorial mardown file ](https://github.com/GoogleCloudPlatform/apigee-samples/blob/main/cloud-logging/docs/cloudshell-tutorial.md)as well

Reply:
OK I've done that update. Have a look?